content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -397,7 +397,6 @@ Each feature gate is designed for enabling/disabling a specific feature:
 - `ProcMountType`: Enables control over ProcMountType for containers.
 - `PVCProtection`: Enable the prevention of a PersistentVolumeClaim (PVC) from
   being deleted when it is still used by any Pod.
-  More details can be found [here](/docs/tasks/administer-cluster/storage-object-in-use-protection/).
```

Comment:
Hello @butuzov . This change seems fine. From what I can see, the content in data/tasks.yml is no longer used.

Reply:
Should I delete `tasks.yml` if its no longer used?

Reply:
@butuzov, no, I don't think so. I would probably recommend removing the task change from this PR and leave the file as is.

Reply:
done.